rpc/rpcserver/server.go

```diff
@@ -1772,10 +1771,10 @@ func (t *ticketbuyerServer) TicketBuyerConfig(ctx context.Context, req *pb.Ticke
 		MaxPriceAbsolute:      config.MaxPriceAbsolute,
 		MaxPriceRelative:      config.MaxPriceRelative,
 		MaxInMempool:          int64(config.MaxInMempool),
-		PoolAddress:           config.PoolAddress,
+		PoolAddress:           config.PoolAddress.String(),
```

Comment:
This will panic when the address is nil.  It must check for this and use the empty string instead.